### ISSUE TITLE
test failure: cannot stat 'mmbiblio.html': No such file or directory

### ISSUE THREAD

**Issue by jkingdon** [metamath/set.mm#2986]
For example, in https://github.com/jkingdon/set.mm/actions/runs/3879557532/jobs/6616849029 it is failing with:

```
# If there's no more recent mmbiblio.html, use this one.
if ! [ -e ../mmbiblio.html ] ; then
  cp -p mmbiblio.html ../
fi
cp: cannot stat 'mmbiblio.html': No such file or directory
Error: Process completed with exit code 1.
```

**Comment by jkingdon** [comment]
The tests for https://github.com/metamath/set.mm/pull/2987 did not fail (is the different in results pull request builds versus just pushing a branch? Some other factor?)

**Comment by david-a-wheeler** [comment]
I believe that's because we've started to pull metamath.exe from GitHub, and that has a different structure. I think that code needs to be tweaked to be more resilient.

**Comment by david-a-wheeler** [comment]
Ugh, there's a bug in the tweaked build-metamath script.

I think this fixes it:
https://github.com/metamath/set.mm/pull/2988

**Comment by david-a-wheeler** [comment]
I merged #2988. If you merge the "develop" branch in, that *should* fix it. Sorry for the trouble!! When I fix one thing, another part goes wonky, but hopefully all options are now accounted for.

**Comment by jkingdon** [comment]
Passed at https://github.com/jkingdon/set.mm/actions/runs/3920720634 so I'll close this issue.
